spec/unit/provider/a2mod/gentoo_spec.rb

```diff
@@ -32,9 +32,11 @@
     end
 
     it 'normalizes parameters' do
-      filetype.expects(:read).returns(%(APACHE2_OPTS="-D FOO -D BAR -D BAR"\n))
-      provider_class.expects(:filetype).returns(filetype)
-
+      # filetype.expects(:read).returns(%(APACHE2_OPTS="-D FOO -D BAR -D BAR"\n))
+      allow(filetype).to receive(:read).and_return(%(APACHE2_OPTS="-D FOO -D BAR -D BAR"\n))
```

Comment:
This should be an expect.

Reply:
Unless you add a `expect(filetype).to have_received(:read)` later on. IIRC there's a rubocop-rspec warning about this.